src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import { useDispatch } from 'react-redux';
+import { Form, Modal, Button, TransitionReplace } from '@edx/paragon';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import FormSwitchGroup from '../../generic/FormSwitchGroup';
+import messages from './messages';
+
+import { closeSettingsModal } from '../data/slice';
+import { appSettingsShape } from '../data/api';
+import { enableDisableApp } from '../data/thunks';
+
+function AppSettingsModal({
+  intl, settings, title, children, onCancel, onApply, enableAppLabel,
+  enableAppHelp, learnMoreURL, learnMoreText,
+}) {
+  const dispatch = useDispatch();
+  const learnMoreLink = (
+    <a href={learnMoreURL}>
+      {learnMoreText}
+              &nbsp;
+      <FontAwesomeIcon icon={faExternalLinkAlt} />
+    </a>
+  );
+  return (
+    <Modal
```

Comment:
Please take a look at: https://edx.github.io/paragon/components/modal/modal-layer/

`ModalLayer` is a much less opinionated modal component in Paragon that will ultimately be replacing the existing `Modal`.  I think I'd prefer to use that for `AppSettingsModal`.  You can see an example of its use in `FullScreenModal` here in frontend-app-course-authoring.  While that one is full page, you could instead just set a container utility class on the `<div role="dialog">` element inside the ModalLayer to get whatever size the mockups call for.

Reply:
Sure, will look into it.